documentation/index.markdown

```diff
@@ -26,6 +26,8 @@ Puppet Server is the next-generation application for managing Puppet agents.
 * **Administrative API**
     * [Environment Cache](./admin-api/v1/environment-cache.markdown)
     * [JRuby Pool](./admin-api/v1/jruby-pool.markdown)
+* **Puppet API**
+    * [Environment Classes](./puppet/v3/environment_classes.markdown)
```

Comment:
@camlow325 You don't need to worry about that, I'll take care of checking the links. FWIW though, I just filed PR #941 with the `static_file_content` endpoint, and I'm putting it in `./puppet-api/v3/static_file_content.md`. So you're close!